AcademyResidentInformationApi.Tests/V1/E2ETests/GetClaimantInformationById.cs

```diff
@@ -23,11 +23,13 @@ public void SetUp()
         public async Task GetClaimantInformationByIdReturnsTheCorrectInformation()
         {
             var claimId = _fixture.Create<int>();
+            var houseId = _fixture.Create<int>();
+            var memberId = _fixture.Create<int>();
             var personRef = _fixture.Create<int>();
             var academyId = $"{claimId}-{personRef}";
```

Comment:
I don't think we need this Academy Id anymore since we don't pass it as part of the URL.

Reply:
@LBHBomotoso and I have decided to have these changes in a separate PR.

Reply:
@LBHBomotoso and I have decided to have these changes in a separate PR.